Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs

```diff
@@ -16,6 +16,9 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+extern alias realm;
```

Comment:
Is there some convention that I am unaware of stating that extern aliases are named with lowercase?